telegram-bot/main.go

```diff
@@ -62,7 +62,7 @@ func help(ID int64) {
 	/irc - Find us on IRC.
 	/xkcd - Get a xkcd comic for you.
 	/dlmeetups - Get the list of upcoming meetups of Delhi/NCR communities.
-	/addmeetup* - Add the details of next OSDC Meetup.` + "\n" + `Format : /addmeetup <Title> <DD/MM/YYYY>
+	/addmeetup* - Add the details of next OSDC Meetup.` + "\n" + `Format : /addmeetup <Title> <DD MM YYYY hh min> <Venue>
```

Comment:
Will we be accepting dates as `DD MM YYYY` or will `DD/MM/YYYY` also work?

Reply:
Made some changes in `addmeetup` function so that we can countinue using the `DD/MM/YYYY` format.
```suggestion
	/addmeetup* - Add the details of next OSDC Meetup.` + "\n" + `Format : /addmeetup <Title> <DD/MM/YYYY> <hh:min> <Venue>
```